docs/installguide/install_all.rst

```diff
@@ -80,13 +89,18 @@ Uninstalling
 Windows
 _______
 
-Uninstall KA Lite from the Control Panel.
-In Windows XP, double-click the "Add or Remove Programs" icon, then choose KA Lite.
-In later version of Windows, click the "Programs and Features" icon, then choose KA Lite.
+1. Uninstall KA Lite from the Control Panel.
+2. In Windows XP, double-click the "Add or Remove Programs" icon, then choose KA Lite.
+3. In later version of Windows, click the "Programs and Features" icon, then choose KA Lite.
 
 Mac OSX
 _______
 
+1. Uninstall KA-Lite Monitor.

```

Comment:
The instructions are unclear in here. What does it mean to 'uninstall ka lite`? Maybe step 1 is actually opening the preferences dialog box first?